### PR TITLE
Fix wrong GitHub actions configuration

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/build-test.yaml [hunk 1]
```diff
@@ -17,11 +17,16 @@ jobs:
         with:
           go-version: '1.17.6'
       - run: GOOS=linux GOARCH=amd64 go build -o artifacts/pipecd ./cmd/pipecd
```
Comment (knanao) [inline .github/workflows/build-test.yaml:19]: I think it better which we create make command and call that command from CI.
However we've already used `build` for bazel, it's up to you whether replace that command or not.

Comment (nghialv) [reply .github/workflows/build-test.yaml:19]: Totally agree about that.
I will refactor it later once confirmed that it works for our case.

**Comment by knanao** [comment]
Nice
/lgtm

**Comment by khanhtc1202** [comment]
Thanks
/approve

**Pull request merged**